examples/mobile/UIComponents/components/controllers/index.html

```diff
@@ -28,69 +28,24 @@ <h1>
 						Checkbox (Image thumbnail)
 					</a>
 				</li>
-				<li class="ui-li-anchor">
-					<a href="button/index.html">
-						Button
-					</a>
-				</li>
-				<li class="ui-li-anchor">
-					<a href="onoffswitch.html">
-						On/Off switch
-					</a>
-				</li>
-				<li class="ui-li-anchor">
-					<a href="master-onoffswitch.html">
-						Master On/Off switch
-					</a>
-				</li>
 				<li class="ui-li-anchor">
 					<a href="radio.html">
 						Radio Button
 					</a>
 				</li>
 				<li class="ui-li-anchor">
-					<a href="floatingactions/index.html">
-						Floating Actions
-					</a>
-				</li>
-				<li class="ui-li-anchor">
-					<a href="textinput.html">
-						Text Field
-					</a>
-				</li>
-				<li class="ui-li-anchor">
-					<a href="textenveloper/index.html">
-						Text Enveloper
-					</a>
-				</li>
-				<li class="ui-li-anchor">
-					<a href="searchinput/index.html">
-						Search Input
-					</a>
-				</li>
-				<li class="ui-li-anchor">
-					<a href="searchbar/index.html">
-						Search Bar
+					<a href="master-onoffswitch.html">
+						Master On/Off switch
 					</a>
 				</li>
 				<li class="ui-li-anchor">
-					<a href="progress/index.html">
-						Progress &amp; Activity
+					<a href="onoffswitch.html">
+						On/Off switch
```

Comment:
On/Off switch is correct?
it is already input Progress in examples/mobile/UIComponents/components/content/index.html file.
are you will remove this file?

Reply:
On/Off switch is just located in the examples/mobile/UIComponents/components/controllers/index.html file.
There isn't in the content example.